go/chat/boxer.go

```diff
@@ -275,26 +275,13 @@ func (b *Boxer) UnboxMessage(ctx context.Context, boxed chat1.MessageBoxed, conv
 				boxed.ClientHeader.TlfPublic))), nil
 	}
 	keyMembersType := b.getEffectiveMembersType(ctx, boxed, conv.GetMembersType())
-	nameInfo, err := CtxKeyFinder(ctx, b.G()).FindForDecryption(ctx,
+	encryptionKey, err := CtxKeyFinder(ctx, b.G()).FindForDecryption(ctx,
 		tlfName, boxed.ClientHeader.Conv.Tlfid, conv.GetMembersType(),
 		conv.IsPublic(), boxed.KeyGeneration, keyMembersType == chat1.ConversationMembersType_KBFS)
 	if err != nil {
 		// Check to see if this is a permanent error from the server
 		return chat1.MessageUnboxed{}, b.detectKBFSPermanentServerError(err, tlfName)
```

Comment:
I suspect any `DecryptionKeyNotFoundError` should be wrapped in `NewPermanentUnboxingError` before returning from `UnboxMessage`. Could add it to `detectKBFSPermanentServerError` (which is due for a rename to kitchen sink).

Reply:
Yeah true, I'll look at that.